forge-gui/res/cardsfolder/upcoming/lumra_bellow_of_the_woods.txt

```diff
@@ -0,0 +1,14 @@
+Name:Lumra, Bellow of the Woods
+ManaCost:4 G G
+Types:Legendary Creature Elemental Bear
+PT:*/*
+K:Vigilance
+K:Reach
+S:Mode$ Continuous | EffectZone$ All | CharacteristicDefining$ True | SetPower$ X | SetToughness$ X | Description$ CARDNAME's power and toughness are each equal to the number of lands you control.
+SVar:X:Count$Valid Land.YouCtrl
+SVar:BuffedBy:Land
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigMill | TriggerDescription$ When CARDNAME enters the battlefield, mill four cards. Then return all land cards from your graveyard to the battlefield tapped.
```

Comment:
TriggerDescription doesn't match fully

Reply:
Cheers, tweaked `enters the battlefield` -> `enters`

Reply:
CARDNAME -> NICKNAME

Reply:
Ahh understood